bip-musig2.mediawiki

```diff
@@ -549,6 +549,15 @@ There are, however, insecure toy schemes with AGM security proofs, but those exp
 In contrast, in MuSig2, all group elements that arise in the protocol are known to the adversary and declared as group element inputs.
 A scheme very similar to MuSig2 and with two-point nonces was independently proven secure in the ROM and AGM by [https://eprint.iacr.org/2020/1245 Alper and Burdges].
 
+== Change Log ==
+
+To help implementors understand updates to this BIP, we attach a version number that resembles ''semantic versioning'' (<code>MAJOR.MINOR.PATCH</code>).
+A change in the <code>MAJOR</code> version indicates that the specification is incompatible with prior versions.
+The <code>MINOR</code> version is increased whenever the inputs or the output of an algorithm changes in a backward compatible way.
+
+* '''0.2.0''' (2022-04-19): Change order of arguments in ''NonceGen'' hash function
```

Comment:
Now a month has passed... (Sorry for that!) But even if we're quick, we won't always merge on the same day. Should we stick to dates here? Should we insert the dates only merging? Should we use commit ids instead? (commit ids are problematic if we PR to the official BIPs repo later but doing it for now wouldn't be too bad and then we could replace by dates later?)

Reply:
Right now it's the date of the commit. This helps finding the commit in the log even without the commit id. If it's not too distant from the merge I think that's fine and simpler than requiring the date to be the same as the merge commit. Bumped the dates.

Reply:
Same here, sounds good and pragmatic :)